etc/scipipe/build_matrix.yaml

```diff
@@ -5,7 +5,7 @@
 # provide yaml anchors internal to this file.
 #
 template:
-  splenv_ref: &splenv_ref '984c9f7'
+  splenv_ref: &splenv_ref 'ed862f4'
```

Comment:
Should change to the master commit hash once it's merged?

Reply:
Yeah, after I merge scipipe_conda_env I will update this repo, lsst/lsst, and lsst/lsstsw with the hash.